crev-lib/src/util/mod.rs

```diff
@@ -15,14 +16,23 @@ pub const APP_INFO: app_dirs::AppInfo = app_dirs::AppInfo {
     author: "Dawid Ciężarkiewicz",
 };
 
+fn get_git_default_editor() -> Result<String> {
+    let cfg = git2::Config::open_default()?;
+    Ok(cfg.get_string("core.editor")?)
+}
+
 fn get_editor_to_use() -> Result<ffi::OsString> {
-    Ok(if let Some(v) = env::var_os("VISUAL") {
-        v
-    } else if let Some(v) = env::var_os("EDITOR") {
-        v
-    } else {
-        "vi".into()
-    })
+    Ok(
+        if let Some(v) = env::var_os("EDITOR") {
```

Comment:
That wouldn't work for me, as I prefer my setting of `VISUAL` over less usable `EDITOR`.

Reply:
I think other tools use `VISUAL` over `EDITOR` in general (e.g. git itself if you don't specify editor), so for these two that's the right order.

I'm not sure how git editor should interact with these. It might be fine even as the first choice if we assume we're copying git's behavior.

Reply:
`VISUAL` is usually checked ahead on `EDITOR`. Historically `EDITOR` was first and then `VISUAL` was introduced for terminals that can support visual output. (`vi` vs `ed`)